namer/namer.cc

```diff
@@ -15,16 +18,1336 @@
 
 using namespace std;
 
+// TODO:
+// - Remove redundancy whenever possible.
+// - ENFORCE that NameFinder never produces errors.
```

Comment:
Flagging some TODOs.

I'm fine leaving TODOs in the codebase, but if you do, I'd very much prefer to have them next to a specific line, instead of at the top of a file.

Reply:
Thank you for pointing this out. This was overlooked during my initial scan of the diff.